i18n/languages.md

```diff
@@ -7,5 +7,5 @@
 - [Italian / Italiano](README.it-it.md) 
 - [Ukranian / Українська](README.ua-ua.md) 
 - [Spanish / Español](./README.es-es.md)
-- [Portuguese / Português](README-PT.md)
+- [Portuguese / Português](README.pt.md)
```

Comment:
This file is renamed above (to match all the other files in the directory)

Reply:
there are multiple references to this file in other readme files. There will be broken links

Reply:
Ah, my tooling generally skips changing files I choose to Ignore, and I chose to ignore the [other README](https://github.com/jsoref/questdb/blob/spell-check/.github/actions/spelling/excludes.txt#L107) files.

Also, in my experience, about half the time translators get annoyed when one changes translations because they like to manage translations themselves (the other half of the time, they get annoyed that one hasn't updated that -- it's nearly a perfect split, so I really can't win).

Anyway, changed.